lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -1618,7 +1618,7 @@ def test_unenrolled_allowed_to_enroll_user(self):
         paid_course = self.create_paid_course()
```

Comment:
In any of these tests we are not validating present of the desired values for `role` and `reason` fields in response returned by APIs. We should have a test to validate it is persisted at the backend store.